### DISCUSSION TITLE
[blocks]: Dashboard template with multi-level sidebar

### DISCUSSION THREAD

**Discussion by daffafikri19** [shadcn-ui/ui#3232] [Blocks Request]
### Description

A dashboard template made with shadcn-ui components with multi-level sidebar menu

### Example

https://i.ibb.co/C5YpS0N/multi-level-sidebar.png

**Comment by monecchi** [comment]
A well crafted example by @salimi-my [here](https://github.com/salimi-my/shadcn-ui-sidebar) using Shadcn ui
